tests/run_tests_headless.sh

```diff
@@ -1,12 +1,12 @@
-#!/bin/bash
+#!/bin/sh
 set -e
 
 # Run headless "acceptance tests" using Xvfb
 # These tests involve booting the game, loading particular saved games, and making assertions
 # about the state of the game universe. Some tests will involve issuing basic commands.
 
 # TODO: When this script is made cross-platform, replace this OS check with a check for Xvfb and other required resources.
-if [[ $OSTYPE == 'msys' ]] || [[ $OS == 'Windows_NT' ]] || [[ $(uname) == 'Darwin' ]]; then
+if [ "$OSTYPE" == 'msys' ] || [ "$OS" == 'Windows_NT' ] || [ "$(uname)" == 'Darwin' ]; then
```

Comment:
Note that you'll want to change the `==` to `=`, as the former doesn't work on some common Bourne shells (such as `dash`).

Reply:
First I forget the `""`, then I forgot the `=`. What am I doing. Thanks!